### ISSUE TITLE
Access token expiry time not set properly

### ISSUE THREAD

**Issue by srikar99** [getinsomnia/insomnia#5084]
### Expected Behavior

When we fetch access token using OAuth 2.0 and client-credentials, the expiry time of the token should be displayed

### Actual Behavior

Currently we only see, **ACCESS TOKEN (EXPIRES JUST NOW)**

![image](https://user-images.githubusercontent.com/15083042/184859687-0607427f-28e0-4ebc-b116-1c1d0fa3b6bf.png)

### Reproduction Steps

1. Go to OAuth 2.0 in the Auth tab.
2. Select `grant_type ` as `client_credentials`
3. Give the appropriate client id an secret.
4. Click on Fetch/Refresh token.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

Version: Insomnia 2022.5.0
Build date: 8/2/2022
OS: Windows_NT x64 10.0.19042
Electron: 19.0.3
Node: 16.14.2
V8: 10.2.154.4-electron.0
Architecture: x64

### Insomnia Version

2022.5.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10.0.19042

### Installation method

download from insomnia.rest

**Comment by filfreire** [comment]
Hi @srikar99, thank you for reporting this!

We were able to reproduce this as well!

We did notice a difference between older version (2021.7.2) and more recent versions (e.g. 2022.4.2 and 2022.5.1), so we might have broken this at some point. It only seems to affect the label though, the timeout time is still the expected one.

On latest versions:
![image](https://user-images.githubusercontent.com/11976836/185142963-e638b890-f765-42a3-88d9-6f3c6d22a43c.png)

On older versions:
![image](https://user-images.githubusercontent.com/11976836/185143081-459b9d07-598f-4eb0-a753-b4689f645eb3.png)
